main_comment_test.go

```diff
@@ -349,6 +349,22 @@ func TestParsePrOptions(t *testing.T) {
 				"mender":         "3.1.x",
 			},
 		},
+		"start pipeline with --pr flags (new syntax)": {
+			StartPipelineComment: "start pipeline --pr mender-connect/pull/88 --pr deviceconnect/pull/12 --pr mender/3.1.x",
+			RepoToPr: map[string]string{
+				"mender-connect": "pull/88/head",
+				"deviceconnect":  "pull/12/head",
+				"mender":         "3.1.x",
+			},
+		},
+		"start pipeline with --pr flags (syntax without 'pull' and 'head')": {
+			StartPipelineComment: "start pipeline --pr mender-connect/88 --pr deviceconnect/12 --pr mender/3.1.x",
+			RepoToPr: map[string]string{
+				"mender-connect": "pull/88/head",
+				"deviceconnect":  "pull/12/head",
+				"mender":         "3.1.x",
+			},
+		},
```

Comment:
Please add a `--pr mender/feature-my-feature-branch` as well, so we have that tested once and for all.

Reply:
Done!